mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -248,7 +248,7 @@ class VizServlet extends ScalatraServlet {
       contentType = "json"
       val viewRegion = ReferenceRegion("chr" + params("ref"), params("start").toLong, params("end").toLong)
       if (VizReads.readsPath.endsWith(".adam")) {
-        val pred: FilterPredicate = ((LongColumn("start") >= viewRegion.start) && (LongColumn("start") <= viewRegion.end))
+        val pred: FilterPredicate = ((LongColumn("end") >= viewRegion.start) && (LongColumn("start") <= viewRegion.end))
```

Comment:
Oh, got it now!  I think the difference between what the two of us are thinking is "overlaps partially" vs. "contains". Your predicate implements _x_ is fully contained in _y_ while mine implements _x_ and _y_ overlap in at least one position. Both are valid interpretations of overlaps; I'm cool with either one.

Also, I think you are right about the "<" vs "<=", but I need to squint at it more. I'll whiteboard it out tomorrow and see if I have convinced myself one way or the other...